wordwrap.go

```diff
@@ -53,6 +53,10 @@ func WrapString(s string, lim uint) string {
 			wordBuf.WriteRune(char)
 
 			if current+uint(spaceBuf.Len()+wordBuf.Len()) > lim && uint(wordBuf.Len()) < lim {
+				if equalLen {
```

Comment:
@liggit the way this algorithm works, if there is a sequence of white spaces that falls or go beyond the `lim` , it replaces that with `\n` and so reduction in length. What this PR did, lets say there is sequence of x white spaces, it replaced them with x-1 white spaces + '\n' so preserving the length.